tlslite/messages.py

```diff
@@ -1680,6 +1680,50 @@ def write(self):
         return self.postWrite(w)
 
 
+class EncryptedExtensions(HandshakeMsg):
+    """Handling of the TLS1.3 Encrypted Extensions message."""
+
+    def __init__(self):
+        super(EncryptedExtensions, self).__init__(
+                HandshakeType.encrypted_extensions)
+        self.extensions = None
+
+    def create(self, extensions):
+        """Set the extensions in the message."""
+        self.extensions = extensions
+
+    def parse(self, parser):
+        """Parse the extensions from on the wire data."""
+        parser.startLengthCheck(3)
+
+        if not parser.getRemainingLength():
+            raise SyntaxError("No list of extensions")
```

Comment:
My point was on a different direction. Couldn't the EE message be empty by itself, and raising an error there would prohibit an otherwise good protocol run?